pkg/sources/github/github.go

```diff
@@ -1514,14 +1519,16 @@ func (s *Source) chunkPullRequestComments(ctx context.Context, repoInfo repoInfo
 	return nil
 }
 
-func (s *Source) scanTargets(ctx context.Context, targets []sources.ChunkingTarget, chunksChan chan *sources.Chunk) error {
+func (s *Source) scanTargets(ctx context.Context, targets []sources.ChunkingTarget, chunksChan chan *sources.Chunk) []*sources.TargetedScanError {
+	var errs []*sources.TargetedScanError
```

Comment:
**Suggestion:** You could replace `[]*sources.TargetedScanError` with `[]error` on both of these lines and avoid the conversion in the parent function.